handbook/product/product_management/direction_template.md

```diff
@@ -0,0 +1,111 @@
+# Vision, Strategy, and Roadmap Template
+
+> ℹ️  The following is a template for use on the pages of specific product & engineering teams, used to record and share the vision, strategy, and roadmap.<br>Copy to your team's page and complete accordingly. Other sources of truth (such as Q3 one-pagers) should have their content integrated here and then be marked as deprecated. Be sure to also replace instances of `ProductArea` with your actual product area name. Unless otherwise specified, each section should be a paragraph or two but you're welcome to use your own judgement if you think a deeper dive is appropriate.
```

Comment:
```suggestion
> ℹ️  The following is a template for use on the pages of specific product & engineering teams, used to record and share the vision, strategy, and roadmap.<br>Copy to your team's page and complete accordingly. Other sources of truth (such as Q3 one-pagers, existing goals pages) should have their content integrated here and then be marked as deprecated. Be sure to also replace instances of `ProductArea` with your actual product area name. Unless otherwise specified, each section should be a paragraph or two but you're welcome to use your own judgement if you think a deeper dive is appropriate.
```